### ISSUE TITLE
Boards G and H missing

### ISSUE THREAD

**Issue by mccrispy** [SteveBallantine/spirit-island-randomiser#28]
This is specific to digital versions only, but as you now support the launcher (thank you for that), it's worth adding the boards that Handelabra added when adding support for Horizons in digital. I know that in the physical game you can't select these boards individually because of the pre-built maps, but in the digital version you can. Perhaps they could be added to the Boards group, with a "digital only" flag/indicator.

Actually, on reflection I realise that boards F, G & H are all problematic. 

- F can only be selected if JE is an added expansion
- G & H can't be added at all

In digital, F, G & H were added with Horizons content (presumably, E will be added with JE when that content finally gets added)

I'm beginning to wonder whether adding Digital DLCs as expansions might provide a better experience (I _know_, we could build presets…).

**Comment by SteveBallantine** [comment]
I've added G and H now. I know some people (me soon to be included) have made thier own copies of these boards that aren't limited to the fixed board layout, so makes sense to just allow them as an option if you have Horizons I think.

I noticed E wasn't in Handelabra's list, but F was, so I assumed that was just a typo.
I've now fixed this, so it will let you launch with board F, but not E. (seems a little random though!)
